### PR TITLE
[FEATURE]  Make a icon with svg

### PR DESCRIPTION
**Pull request by ramonxm**
## Description

Realizando a otimização do SVG e criando os assets com jsx. 

## Developer Checks

- [X] PR title & commits adhere to [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/)
- [X] PR is targeting correct branch, is up-to-date, & no merge conflicts
- [X] Tested on browser device

### CODE REVIEW

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Comment by ramonxm** [comment]
[Criação de componente Logo.](https://trello.com/c/ja9CCb4e/13-cria%C3%A7%C3%A3o-de-componente-logo)

File: src/assets/svg/Logo.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { LogoProps } from "./types";
+
+const Logo = ({ width = "160", height = "72" }: LogoProps) => (
+  <svg
+    width={width}
```
Comment (jonataspinto) [inline src/assets/svg/Logo.tsx:5]: Que tal usarmos viewBox? no lugar de passar esses tamanhos para o svg.

Comment (ramonxm) [reply src/assets/svg/Logo.tsx:5]: top

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Pull request merged**